### PR TITLE
update waas-restart.md, issue #1841

### PR DESCRIPTION
**Pull request by MaratMussabekov**
change description of auoptions value 2, source [How to configure automatic updates by using Group Policy or registry settings](https://support.microsoft.com/en-ca/help/328010/how-to-configure-automatic-updates-by-using-group-policy-or-registry-s)

### CODE REVIEW

File: windows/deployment/update/waas-restart.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ The following tables list registry values that correspond to the Group Policy se
 | --- | --- | --- |
 | AlwaysAutoRebootAtScheduledTime | REG_DWORD | 0: disable automatic reboot after update installation at scheduled time</br>1: enable automatic reboot after update installation at ascheduled time |
 | AlwaysAutoRebootAtScheduledTimeMinutes | REG_DWORD | 15-180: set automatic reboot to occur after given minutes |
-| AUOptions | REG_DWORD | 2: notify for download and automatically install updates</br>3: automatically download and notify for instllation of updates</br>4: Automatically download and schedule installation of updates</br>5: allow the local admin to configure these settings</br>**Note:** To configure restart behavior, set this value to **4** |
+| AUOptions | REG_DWORD | 2: notify for download and notify for instllation of updates</br>3: automatically download and notify for instllation of updates</br>4: Automatically download and schedule installation of updates</br>5: allow the local admin to configure these settings</br>**Note:** To configure restart behavior, set this value to **4** |
```
Comment (mypil) [inline windows/deployment/update/waas-restart.md]:
```suggestion
| AUOptions | REG_DWORD | 2: notify for download and notify for installation of updates</br>3: automatically download and notify for installation of updates</br>4: Automatically download and schedule installation of updates</br>5: allow the local admin to configure these settings</br>**Note:** To configure restart behavior, set this value to **4** |
```

Comment (mypil) [reply windows/deployment/update/waas-restart.md]: @MaratMussabekov - there was a typo on the PR you created for issue #1841. I made a copy/edit review so please assist to merge this new commit. Thanks.

**Comment by MaratMussabekov** [comment]
Dear @mypil , your suggestion was applied, thank you very much!

**Comment by illfated** [comment]
You may want to rebase your PR branch on the master branch, in order to resolve the file conflict.

**Comment by mypil** [comment]
@MaratMussabekov - please check the comment from illfated. Thank you.

**Comment by MaratMussabekov** [comment]
created new PR - https://github.com/MicrosoftDocs/windows-itpro-docs/pull/2994

**Pull request closed without merging**